lib/activerecord-mysql-structure/active_record/structure_sql_sanitizer.rb

```diff
@@ -86,5 +103,32 @@ def sort_columns(lines)
 
       sorted_lines
     end
+
+    def sort_indices(lines)
+      indices = []
+      sorted_lines = []
+
+      lines.each do |line|
+        match = INDEX_REGEX.match(line)
+        if match
+          # Ensure trailing commas on each line
+          sanitized_index = "#{match[1]},#{match[3]}"
+          # Remove redundant, noisy default index type specifier
+          # TODO: this is not safe on tables using the Memory/Heap nor NDB engines
```

Comment:
Since mysql puts the engines in the sql setup, could we raise in this case and stop the danger from happening?

Reply:
As in, crash with an error message any time a line matches `/ENGINE=(?!InnoDB|MyISAM)/` ?

Reply:
Yeah if we know its a problem and we're currently not using any, seems good to me to let someone know about it?

Reply:
Sounds like a plan to me